app/src/test/java/io/apicurio/registry/DataUpgradeTestProfile.java

```diff
@@ -0,0 +1,16 @@
+package io.apicurio.registry;
+
+import io.quarkus.test.junit.QuarkusTestProfile;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class DataUpgradeTestProfile implements QuarkusTestProfile {
+
+    @Override
+    public Map<String, String> getConfigOverrides() {
+        return Collections.singletonMap("apicurio.upgrade.file.location",
+                getClass().getResource("upgrade/v2_export.zip").toExternalForm());
```

Comment:
I like this just fine but what do you think about putting the import+migration logic directly into the v2 "import" endpoint?  That way end users don't have to mess with the deployment in any way.

Reply:
I don't dislike it, my biggest concern is that it's more error prone. If you use the current implementation it means you really know what you're doing. That said, it's not a strong opinion, so I'm open to change it. @jsenko thoughts?

Reply:
I would only suggest it if we **also** add a check for the manifest file - to verify that the ZIP file is really a v2 export.